src/main/java/mate/academy/spring/mapper/impl/request/UserRequestMapper.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.mapper.impl.request;
+
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserRequestMapper implements DtoRequestMapper<UserRequestDto, User> {
+    @Override
+    public User fromDto(UserRequestDto userRequestDto) {
```

Comment:
bad naming "fromDto" to what?

Reply:
name of this method we get from Generic super class, so I can't change it